build.gradle

```diff
@@ -43,6 +43,7 @@ allprojects{
 
 repositories {
     mavenCentral()
+    maven { url "https://jitpack.io" }
```

Comment:
without this the kxml dependency from https://github.com/dimagi/commcare-core/blob/gradleUpdate/build.gradle#L70 doesn't get resolved.